### ISSUE TITLE
Jaws does not read Show/Hide columns dialog contents on Chrome.Check/Uncheck checkbox with spacebar closes the dialog.

### ISSUE THREAD

**Issue by sumegha26** [vmware/clarity#4750]
## Describe the bug

Jaws does not read Show/Hide columns dialog contents on Chrome.Check/Uncheck checkbox with spacebar closes the dialog.
 
Jaws does not read the label associated with checkbox when checkbox on Show/hide columns dialog is focused, it reads `Show or Hide columns` label on each checkbox focus

Check/Uncheck the checkbox with spacebar closes the dialog.

## How to reproduce

1.  Go to https://clarity.design/documentation/datagrid/hide-show on chrome
2.  Turn on Jaws
2.  Press tab and focus the `Show or Hide columns button`
3.  Press enter dialog gets open
4.  Focus first checkbox, Jaws does not label associated with checkbox
5.  Press space bar to check/uncheck the checkbox, dialog gets closed.

## Expected behavior

Jaws should read label associated with checkboxes and check/uncheck should work with space bar and dialog should not be closed.

## Versions

**App**

- Angular: 9
- Clarity:  3.x

**Device:**

- Type: Desktop
- OS: Windows
- Browser: Chrome
- Version: latest

## Additional notes

Given scenario works on IE11 with Jaws.

-  Adding role = `application` on Show/Hide columns popover works with Chrome + Jaws ( flaw - Jaws reads Show Columns button  as `out of table`)

JFYI, **Chrome + Jaws** combination works fine on **Clarity 1** - https://v1.clarity.design/datagrid/hide-show

**Comment by Jinnie** [comment]
@chlane do we want to cover the Chrome + JAWS scenario?

**Comment by hippee-lee** [comment]
As of last year, supported combinations were:

1. macOS: VoiceOVer + Safari
2. windows: Firefox + NVDA

As far as I know, that hasn't changed.

**Comment by chlane** [comment]
That has not changed.

**Comment by sumegha26** [comment]
FYI, @Jinnie  **Chrome + Jaws** combination works fine on **Clarity 1** - https://v1.clarity.design/datagrid/hide-show

**Comment by chlane** [comment]
We don't have to fix it from an MVP perspective because the[ Clarity VPAT](https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/product/vpat/vmware-vpat-clarity-311-may-2020.pdf) says Clarity is best used with Safari/Firefox or NVDA/Chrome. But of course we would never discourage this fix either.

**Comment by sumegha26** [comment]
@chlane , @hippee-lee  any updates on Show Columns accessibility for Chrome ?

**Comment by chlane** [comment]
@sumegha26 As I mentioned in my previous comment Clarity doesn't support Chrome + JAWS.

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues.Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
